docs/content/basic/installing.rst

```diff
@@ -28,12 +28,14 @@ Installing SimPEG
 Conda Forge
 -----------
 
-You can install SimPEG using the `conda package manager <https://conda.io/>`_ that comes with the Anaconda distribution:
+You can install SimPEG using the `conda package manager <https://conda.io/>`_ that
+comes with the Anaconda distribution:
 
 .. code::
 
     conda install SimPEG --channel conda-forge
 
+Installing through `conda` is our recommended method of installation.
 
```

Comment:
People seem to get frustrated with the speed of conda, for good reason. By teaching and generally I started to recommend mamba by default. And in particular `mambaforge`, as you then have your environment already setup to always use `--channel conda-forge`.
https://github.com/conda-forge/miniforge#mambaforge

I don't know if that is something that could be mentioned here.

Reply:
I think it is worthwhile dropping a link to the `mamba` project here in a ``.. note::` section.
Also, I don't think `mamba` defaults to having the `-c conda-forge` option always on anymore. I've noticed this on a few of the more recent installs I've done using it. It also makes sense considering they are billing it as a "drop-in" replacement for `conda`.

Reply:
No, `mamba` does not have it by default I think. But if you install Python using `mambaforge`, it makes the necessary changes so that you do have the `conda-forge` channel.

Reply:
Ah yes.